pkg/platform/kube/functionres/lazy.go

```diff
@@ -2090,9 +2090,10 @@ func (lc *lazyClient) populateDeploymentContainer(ctx context.Context,
 				Path: http.InternalHealthPath,
 			},
 		},
-		InitialDelaySeconds: 1,
+		InitialDelaySeconds: 5,
```

Comment:
Won't increasing the delay affect the `readinessTimeout` in `waitAvailable`?

Reply:
Not really. the wait and readiness timeout will still hold for the specific time it takes, only thing is that readiness will not kick in ASAP but wait a bit. the delay is still a small number comparing to readiness timeout / wait available. (5 seconds vs 60s+).
the reason that has changed is mainly to avoid early readiness probe on function that still in its initialization phase.